### PR TITLE
toolbox overview added, along with new content for the index.md file

### PR DESCRIPTION
**Pull request by londoncalling**
Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by londoncalling** [comment]
@SvenDowideit @moxiegirl please take a look at this overview for Toolbox, along with the content I added for the index.md page. Let me know if it LGTYou.

**Comment by moxiegirl** [comment]
LGTM

**Comment by SvenDowideit** [comment]
@moxiegirl @londoncalling should toolbox.md be called that, or should it be `overview.md` ?

File: docs/toolbox.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+* Oracle VirtualBox
+
+## Ready to get started?
+
+<a href="https://www.docker.com/products/docker-toolbox" target="_blank">Download Docker Toolbox</a> from the product page.
+
+## Next Steps
+
+* <a href="https://docs.docker.com/mac/" target="_blank">Get Started with Docker for Mac OS X</a>
```
Comment (SvenDowideit) [inline docs/toolbox.md]: should these links really be annoying popups?

and should they be to markdown files.

**Comment by londoncalling** [comment]
@moxiegirl @SvenDowideit fixes made to links and file re-name, please let me know if this is ready to go or if you need other changes.

**Comment by SvenDowideit** [comment]
LGTM

**Comment by londoncalling** [comment]
@jeffdm if this looks good to you, please merge last fixes

**Pull request merged**